Lib/distutils/sysconfig.py

```diff
@@ -93,14 +93,11 @@ def get_python_inc(plat_specific=0, prefix=None):
             # the build directory may not be the source directory, we
             # must use "srcdir" from the makefile to find the "Include"
             # directory.
-            base = _sys_home or project_base
             if plat_specific:
-                return base
-            if _sys_home:
-                incdir = os.path.join(_sys_home, 'Include')
+                return _sys_home or project_base
```

Comment:
Sure, but just doing what was already there (and note that they are used "bare" everywhere else in the file).